tests/contrib/sensors/test_redis_cluster_sensor.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+import unittest
+from airflow import DAG
+from airflow import configuration
+from airflow.contrib.hooks.redis_cluster_hook import RedisClusterHook
+from airflow.contrib.sensors.redis_cluster_key_sensor import RedisClusterKeySensor
+from airflow.utils import timezone
+
+DEFAULT_DATE = timezone.datetime(2017, 1, 1)
+
+
+class TestRedisClusterSensor(unittest.TestCase):
+
+    def setUp(self):
+        configuration.load_test_config()
+        args = {
+            'owner': 'airflow',
+            'start_date': DEFAULT_DATE
+        }
+
+        self.dag = DAG('test_dag_id', default_args=args)
+        self.sensor = RedisClusterKeySensor(
+            task_id='test_task',
+            redis_cluster_conn_id='redis_cluster_default',
+            dag=self.dag,
+            key='test_key'
+        )
+
+    def test_poke(self):
+        hook = RedisClusterHook(
+            redis_cluster_conn_id='redis_cluster_default',
+            skip_full_coverage_check=True
+        )
+        redis_cluster = hook.get_conn()
+        redis_cluster.set('test_key', 'test_value')
```

Comment:
Same here: I'd prefer us to mock the return from the redis_cluster object instead.